addon-test-support/@ember/test-helpers/dom/fire-event.js

```diff
@@ -15,15 +15,6 @@ const MOUSE_EVENT_TYPES = [
 ];
 const FILE_SELECTION_EVENT_TYPES = ['change'];
 
-/**
-  @method fireEvent
-  @param {Element} element
-  @param {String} type
-  @param {Object} [options]
-  @returns {Event}
-
-  @private
-*/
```

Comment:
Why did you remove these? They might still be useful for type info

Reply:
Hmm, unless we are going to type check ember-test-helpers internally, I don't think this provides any value. This entire file is private and only used to expose the actual public API's which _do_ have valid type information...

Reply:
Some IDEs are able to parse these comments and provide hints. Might be useful for future contributors.

Reply:
OK, will bring it back for the export (but not the inner / private helper functions)